nohost/lib/async.js

```diff
@@ -1,90 +1,1122 @@
 /*!
```

Comment:
The changes in this file look wrong, like you've reverted things.  Are you rebased to master?